WChat/src/app/screens/groups/index.js

```diff
@@ -1,26 +1,39 @@
 import React from 'react';
-import groups from '../../../../resources/groups.json';
 import Group from './components/Group';
 import FilterAddBar from '../../components/FilterAddBar';
 import {
   View,
   FlatList,
-  StyleSheet
+  StyleSheet, 
+  Text,
+  TouchableOpacity
 } from 'react-native';
+import { connect } from 'react-redux'
 import styles from './styles'
-const createGroup = ({item}) => <Group group={item}/>;
+
+const createGroup = (navigation) => ({item}) => 
+  <TouchableOpacity 
+    onPress={() => {
+      navigation.navigate('GroupChat', {group: item})}}>
```

Comment:
in the next PR, implement this for navigation 😄  
https://reactnavigation.org/docs/guides/redux